wasmbinding/stargate_whitelist_test.go

```diff
@@ -0,0 +1,114 @@
+package wasmbinding_test
+
+import (
+	"encoding/hex"
+	"fmt"
+	"testing"
+	"time"
+
+	proto "github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/osmosis-labs/osmosis/v11/app"
+	"github.com/osmosis-labs/osmosis/v11/wasmbinding"
+)
+
+type StargateTestSuite struct {
+	suite.Suite
+
+	ctx sdk.Context
+	app *app.OsmosisApp
+}
+
+func (suite *StargateTestSuite) SetupTest() {
+	suite.app = app.Setup(false)
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{Height: 1, ChainID: "osmosis-1", Time: time.Now().UTC()})
+}
+
+func TestStargateTestSuite(t *testing.T) {
+	suite.Run(t, new(StargateTestSuite))
+}
+
+func (suite *StargateTestSuite) TestDeterministicJsonMarshal() {
```

Comment:
Looking at this again - I think we should also consider testing invalid inputs, not only happy path

Reply:
Added test case!